tests/rootfs.sh

```diff
@@ -0,0 +1,149 @@
+#!/bin/sh
+
+set -eu
+
+ERROR=1
+OK=0
+MY_NAME=$(basename -- "$0")
+ROOTFS_DIRECTORY="./rootfs"
+TEST_IMAGE_FILE_PATH="${ROOTFS_DIRECTORY}/tmp/test.img"
+TEST_EMPTY_MTAB_FILE="${ROOTFS_DIRECTORY}/etc/mtab"
+
+test_setup()
+{
+    if [ ! -f "${ROOTFS_DIRECTORY}/usr/bin/qemu-arm-static" ]
+    then
+        cp "$(command -v qemu-arm-static)" "${ROOTFS_DIRECTORY}/usr/bin/"
+    fi
+
+    dd if=/dev/zero of="${TEST_IMAGE_FILE_PATH}" bs=2M count=64 >/dev/null 2>&1 || return ${ERROR}
```

Comment:
while we are only creating a very small partition for this test (32 MB?) a block size of 2M is not super ideal.
While there is no 'right' answer here, and bigger is not always better, 2M is relatively small.

I suggest using a single 32M block in this case, which tells us immediately we have a 32mb image :)
secondly, why capture the output; let it go! it's a test after all :) if this fails, we don't even know why, just that test_setup failed.

Finally haven't used return in ages and thus forgot it's true meaning :p (you can't actually return anything, you just set the exit status of the function call) so in that case, the use seems appropriate, but as before, just return 1 in case of error.

Reply:
Yes removed the whole OK ERROR stuff. 
But 32MB does not work for f2fs it needs > 100MB
Therefore made 4 x 32MB